web-client/src/presenter/actions/addPenaltyInputAction.js

```diff
@@ -9,7 +9,7 @@ import { state } from 'cerebral';
  * @returns {void}
  */
 export const addPenaltyInputAction = ({ get, store }) => {
-  const penalties = get(state.modal.penalties) || [];
+  const penalties = get(state.modal.penalties);
```

Comment:
Is `state.modal.penalties` guaranteed to exist? From what I can tell, if penalties is null or undefined, we'll get an error on line 13. Or does cerebral guarantee that `get()` returns something that can have its length checked?

Reply:
Personally I would prefer to not treat something as guaranteed because the current sequence of events that leads to executing the code makes it true. The other changes here (attachments and hearings) seem to be safe, but this one, if someone makes a change that would make us of this function without having called `setDefaultPenaltiesAction` and doesn't write a test case that would catch it, then we could error out just because we're checking length.